src/metabase/driver/sync.clj

```diff
@@ -56,12 +61,13 @@
    (let [{prop-name :name} (driver.u/find-schema-filters-prop (driver.u/database->driver database))]
      (db-details->schema-filter-patterns prop-name database)))
   ([prop-nm {db-details :details :as _database}]
-   (let [schema-filter-type     (get db-details (keyword (str prop-nm "-type")))
-         schema-filter-patterns (get db-details (keyword (str prop-nm "-patterns")))]
-     (case schema-filter-type
-       "exclusion" [nil schema-filter-patterns]
-       "inclusion" [schema-filter-patterns nil]
-       [nil nil]))))
+   (when prop-nm
```

Comment:
not really part of this but for drivers like H2 which don't support schema filtering properties it would look for `:-type` and `:-patterns` keys which seemed silly, I changed it so it just skips this whole block of code instead